v4/weaviate/graphql/rawgql.go

```diff
@@ -0,0 +1,29 @@
+package graphql
+
+import (
+	"context"
+
+	"github.com/semi-technologies/weaviate/entities/models"
+)
+
+// Raw for accepting a prebuilt query from the user
+type Raw struct {
+	connection rest
+	query      string
+}
+
+// Do execute the GraphQL query
+func (gql *Raw) Do(ctx context.Context) (*models.GraphQLResponse, error) {
+	return runGraphQLQuery(ctx, gql.connection, gql.query)
```

Comment:
to be concise I would use `build()` method in `runGraphQLQuery` instead of `qql.query`